<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ICHCConditionPage</class>
 <widget class="QFrame" name="ICHCConditionPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Conditions</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <item row="1" column="1">
       <widget class="ICComboBox" name="subComboBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>123</width>
          <height>38</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>123</width>
          <height>48</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <item>
         <property name="text">
          <string>Sub-1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Sub-2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Sub-3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Sub-4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Sub-5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Bad Product</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Sampling</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Try Product</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0" colspan="4">
       <layout class="QGridLayout" name="gridLayout">
<<<<<<< HEAD
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Check Point</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="defectiveProductsBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>20</width>
            <height>0</height>
           </size>
          </property>
=======
        <property name="horizontalSpacing">
         <number>2</number>
        </property>
        <item row="6" column="2">
         <widget class="ICLineEditWithVirtualNumericKeypad" name="moldCountEdit">
          <property name="text">
           <string notr="true">0</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QCheckBox" name="tryProductBox">
>>>>>>> 83a748a... 1.添加吸3吸4
          <property name="styleSheet">
           <string notr="true">::indicator {
     width: 32px;
     height: 32px;
 }

::indicator:unchecked 
{
     image: url(:/resource/check_unchecked.png);
 }
::indicator:checked 
{
     image: url(:/resource/check_checked.png);
 }

QRadioButton::indicator:checked:disabled
{
 image: url(:/resource/check_checked_disable.png);
}

QRadioButton::indicator:unchecked:disabled
{
 image: url(:/resource/check_unchecked_disable.png);
}</string>
          </property>
          <property name="text">
<<<<<<< HEAD
           <string>Defective Products</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="samplingBox">
=======
           <string>Try Product</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0" colspan="2">
         <widget class="QCheckBox" name="productCountBox">
>>>>>>> 83a748a... 1.添加吸3吸4
          <property name="styleSheet">
           <string notr="true">::indicator {
     width: 32px;
     height: 32px;
 }

::indicator:unchecked 
{
     image: url(:/resource/check_unchecked.png);
 }
::indicator:checked 
{
     image: url(:/resource/check_checked.png);
 }

QRadioButton::indicator:checked:disabled
{
 image: url(:/resource/check_checked_disable.png);
}

QRadioButton::indicator:unchecked:disabled
{
 image: url(:/resource/check_unchecked_disable.png);
}</string>
          </property>
          <property name="text">
<<<<<<< HEAD
           <string>Sampling</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QCheckBox" name="fixture4Box">
=======
           <string>Product Count</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Check Point</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="fixture1Box">
>>>>>>> 83a748a... 1.添加吸3吸4
          <property name="styleSheet">
           <string notr="true">::indicator {
     width: 32px;
     height: 32px;
 }

::indicator:unchecked 
{
     image: url(:/resource/check_unchecked.png);
 }
::indicator:checked 
{
     image: url(:/resource/check_checked.png);
 }

QRadioButton::indicator:checked:disabled
{
 image: url(:/resource/check_checked_disable.png);
}

QRadioButton::indicator:unchecked:disabled
{
 image: url(:/resource/check_unchecked_disable.png);
}</string>
          </property>
          <property name="text">
           <string>Fixture-1</string>
          </property>
         </widget>
        </item>
<<<<<<< HEAD
        <item row="2" column="0">
         <widget class="QCheckBox" name="x044checkBox">
=======
        <item row="1" column="1">
         <widget class="QCheckBox" name="fixture3Box">
>>>>>>> 83a748a... 1.添加吸3吸4
          <property name="styleSheet">
           <string notr="true">::indicator {
     width: 32px;
     height: 32px;
 }

::indicator:unchecked 
{
     image: url(:/resource/check_unchecked.png);
 }
::indicator:checked 
{
     image: url(:/resource/check_checked.png);
 }

QRadioButton::indicator:checked:disabled
{
 image: url(:/resource/check_checked_disable.png);
}

QRadioButton::indicator:unchecked:disabled
{
 image: url(:/resource/check_unchecked_disable.png);
}</string>
          </property>
          <property name="text">
<<<<<<< HEAD
           <string>X044</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QCheckBox" name="fixture1Box">
=======
           <string>Fixture-3</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QCheckBox" name="defectiveProductsBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>20</width>
            <height>0</height>
           </size>
          </property>
>>>>>>> 83a748a... 1.添加吸3吸4
          <property name="styleSheet">
           <string notr="true">::indicator {
     width: 32px;
     height: 32px;
 }

::indicator:unchecked 
{
     image: url(:/resource/check_unchecked.png);
 }
::indicator:checked 
{
     image: url(:/resource/check_checked.png);
 }

QRadioButton::indicator:checked:disabled
{
 image: url(:/resource/check_checked_disable.png);
}

QRadioButton::indicator:unchecked:disabled
{
 image: url(:/resource/check_unchecked_disable.png);
}</string>
          </property>
          <property name="text">
<<<<<<< HEAD
           <string>Fixture-1</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QCheckBox" name="sucker1Box">
=======
           <string>Defective Products</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QCheckBox" name="fixture4Box">
>>>>>>> 83a748a... 1.添加吸3吸4
          <property name="styleSheet">
           <string notr="true">::indicator {
     width: 32px;
     height: 32px;
 }

::indicator:unchecked 
{
     image: url(:/resource/check_unchecked.png);
 }
::indicator:checked 
{
     image: url(:/resource/check_checked.png);
 }

QRadioButton::indicator:checked:disabled
{
 image: url(:/resource/check_checked_disable.png);
}

QRadioButton::indicator:unchecked:disabled
{
 image: url(:/resource/check_unchecked_disable.png);
}</string>
          </property>
          <property name="text">
           <string>Fixture-4</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="fixture2Box">
          <property name="styleSheet">
           <string notr="true">::indicator {
     width: 32px;
     height: 32px;
 }

::indicator:unchecked 
{
     image: url(:/resource/check_unchecked.png);
 }
::indicator:checked 
{
     image: url(:/resource/check_checked.png);
 }

QRadioButton::indicator:checked:disabled
{
 image: url(:/resource/check_checked_disable.png);
}

QRadioButton::indicator:unchecked:disabled
{
 image: url(:/resource/check_unchecked_disable.png);
}</string>
          </property>
          <property name="text">
           <string>Fixture-2</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QCheckBox" name="x043checkBox">
          <property name="styleSheet">
           <string notr="true">::indicator {
     width: 32px;
     height: 32px;
 }

::indicator:unchecked 
{
     image: url(:/resource/check_unchecked.png);
 }
::indicator:checked 
{
     image: url(:/resource/check_checked.png);
 }

QRadioButton::indicator:checked:disabled
{
 image: url(:/resource/check_checked_disable.png);
}

QRadioButton::indicator:unchecked:disabled
{
 image: url(:/resource/check_unchecked_disable.png);
}</string>
          </property>
          <property name="text">
           <string>X043</string>
          </property>
         </widget>
        </item>
<<<<<<< HEAD
        <item row="3" column="1">
         <widget class="QCheckBox" name="fixture2Box">
=======
        <item row="1" column="2">
         <widget class="QCheckBox" name="sucker1Box">
>>>>>>> 83a748a... 1.添加吸3吸4
          <property name="styleSheet">
           <string notr="true">::indicator {
     width: 32px;
     height: 32px;
 }

::indicator:unchecked 
{
     image: url(:/resource/check_unchecked.png);
 }
::indicator:checked 
{
     image: url(:/resource/check_checked.png);
 }

QRadioButton::indicator:checked:disabled
{
 image: url(:/resource/check_checked_disable.png);
}

QRadioButton::indicator:unchecked:disabled
{
 image: url(:/resource/check_unchecked_disable.png);
}</string>
          </property>
          <property name="text">
<<<<<<< HEAD
           <string>Fixture-2</string>
=======
           <string>Sucker-1</string>
>>>>>>> 83a748a... 1.添加吸3吸4
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QCheckBox" name="sucker2Box">
          <property name="styleSheet">
           <string notr="true">::indicator {
     width: 32px;
     height: 32px;
 }

::indicator:unchecked 
{
     image: url(:/resource/check_unchecked.png);
 }
::indicator:checked 
{
     image: url(:/resource/check_checked.png);
 }

QRadioButton::indicator:checked:disabled
{
 image: url(:/resource/check_checked_disable.png);
}

QRadioButton::indicator:unchecked:disabled
{
 image: url(:/resource/check_unchecked_disable.png);
}</string>
          </property>
          <property name="text">
           <string>Sucker-2</string>
          </property>
         </widget>
        </item>
<<<<<<< HEAD
        <item row="4" column="0">
         <widget class="QCheckBox" name="tryProductBox">
=======
        <item row="5" column="0">
         <widget class="QCheckBox" name="samplingBox">
>>>>>>> 83a748a... 1.添加吸3吸4
          <property name="styleSheet">
           <string notr="true">::indicator {
     width: 32px;
     height: 32px;
 }

::indicator:unchecked 
{
     image: url(:/resource/check_unchecked.png);
 }
::indicator:checked 
{
     image: url(:/resource/check_checked.png);
 }

QRadioButton::indicator:checked:disabled
{
 image: url(:/resource/check_checked_disable.png);
}

QRadioButton::indicator:unchecked:disabled
{
 image: url(:/resource/check_unchecked_disable.png);
}</string>
          </property>
          <property name="text">
<<<<<<< HEAD
           <string>Try Product</string>
=======
           <string>Sampling</string>
>>>>>>> 83a748a... 1.添加吸3吸4
          </property>
         </widget>
        </item>
        <item row="4" column="1">
<<<<<<< HEAD
         <widget class="QCheckBox" name="fixture3Box">
=======
         <widget class="QCheckBox" name="x044checkBox">
>>>>>>> 83a748a... 1.添加吸3吸4
          <property name="styleSheet">
           <string notr="true">::indicator {
     width: 32px;
     height: 32px;
 }

::indicator:unchecked 
{
     image: url(:/resource/check_unchecked.png);
 }
::indicator:checked 
{
     image: url(:/resource/check_checked.png);
 }

QRadioButton::indicator:checked:disabled
{
 image: url(:/resource/check_checked_disable.png);
}

QRadioButton::indicator:unchecked:disabled
{
 image: url(:/resource/check_unchecked_disable.png);
}</string>
          </property>
          <property name="text">
<<<<<<< HEAD
           <string>Fixture-3</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QCheckBox" name="productCountBox">
=======
           <string>X044</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QCheckBox" name="sucker3Box">
          <property name="styleSheet">
           <string notr="true">::indicator {
     width: 32px;
     height: 32px;
 }

::indicator:unchecked 
{
     image: url(:/resource/check_unchecked.png);
 }
::indicator:checked 
{
     image: url(:/resource/check_checked.png);
 }

QRadioButton::indicator:checked:disabled
{
 image: url(:/resource/check_checked_disable.png);
}

QRadioButton::indicator:unchecked:disabled
{
 image: url(:/resource/check_unchecked_disable.png);
}</string>
          </property>
          <property name="text">
           <string>Sucker-3</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QCheckBox" name="sucker4Box">
>>>>>>> 83a748a... 1.添加吸3吸4
          <property name="styleSheet">
           <string notr="true">::indicator {
     width: 32px;
     height: 32px;
 }

::indicator:unchecked 
{
     image: url(:/resource/check_unchecked.png);
 }
::indicator:checked 
{
     image: url(:/resource/check_checked.png);
 }

QRadioButton::indicator:checked:disabled
{
 image: url(:/resource/check_checked_disable.png);
}

QRadioButton::indicator:unchecked:disabled
{
 image: url(:/resource/check_unchecked_disable.png);
}</string>
          </property>
          <property name="text">
           <string>Sucker-4</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1" colspan="2">
         <widget class="ICLineEditWithVirtualNumericKeypad" name="moldCountEdit">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Use Macro</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Return Step</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="ICLineEditWithVirtualNumericKeypad" name="returnLineEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>123</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>123</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ICLineEditWithVirtualNumericKeypad</class>
   <extends>QLineEdit</extends>
   <header>iclineeditwithvirtualnumerickeypad.h</header>
  </customwidget>
  <customwidget>
   <class>ICComboBox</class>
   <extends>QComboBox</extends>
   <header>iccombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
